src/test/unit/sync_http/test_sync_hook.py

```diff
@@ -502,6 +503,27 @@ def lambda_test_function(event, context):
     assert event["info"]["httpInfo"]["host"] == "www.google.com"
 
 
+def test_wrapping_requests_times(monkeypatch):
+    @lumigo_tracer()
+    def lambda_test_function(event, context):
+        start_time = time.time() * 1000
+        requests.get("https://www.google.com")
+        return start_time
+
+    # add delay to the connection establishment process
+    original_getaddrinfo = socket.getaddrinfo
+    monkeypatch.setattr(
+        socket,
+        "getaddrinfo",
+        lambda *args, **kwargs: time.sleep(0.1) or original_getaddrinfo(*args, **kwargs),
```

Comment:
I think it should be extracted to a function, I don't think we should combine lines by putting `or` between them, makes it hard to read.